include/swift/AST/Decl.h

```diff
@@ -211,8 +211,11 @@ struct OverloadSignature {
   /// Whether this is an instance member.
   bool IsInstanceMember = false;
```

Comment:
These could be bitfields for efficiency (I know this isn't part of your change but it might be easy to change it)

Reply:
Good idea, thanks